Cargo.toml

```
@@ -13,9 +13,13 @@ keywords = ["linear-programming", "optimization", "math", "solver"]
 
 [build-dependencies]
 bindgen = "0.63.0"
-cmake = "0.1.49"
+cmake = { version = "0.1.49", optional = true }
+pkg-config = { version = "0.3.27", optional = true }
 
 [features]
+default = ["build", "highs_release"]
+discover = ["pkg-config"]
+build = ["cmake"]
```

Don't we need `dep:` here ?

```suggestion
discover = ["dep:pkg-config"]
build = ["dep:cmake"]
```

https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

---

We don't strictly need it. According to the page you linked, this syntax is new in Rust 1.60.

> Note: The dep: syntax is only available starting with Rust 1.60. Previous versions can only use the implicit feature name.

It used to be the case you'd just use the dependencey name (and that also still works). If your MSRV is 1.60 or later anyway, I'd agree it's reasonable to use the new, more explicit syntax, it was just new to me.